Datastore/tests/System/LookupTest.php

```diff
@@ -59,7 +59,7 @@ public function testLookupWithReadTime(DatastoreClient $client)
         $this->assertEquals($person['lastName'], $newLastName);
 
         // Person lastName should be the lastName BEFORE update
-        $person = $client->lookup($key, ['readTime' => $time]);
+        $person = $client->lookup($key, ['readTime' => $time->formatForApi()]);
```

Comment:
We shouldn't be asking users to call this, I don't think, also this wouldn't match the type of `Timestamp` in the method docblocks.

Reply:
my bad, I had missed it.